### DISCUSSION TITLE
Is there a UTM SE for TestFlight? And if so, how could I

### DISCUSSION THREAD

**Discussion by Sahelalemi** [utmapp/UTM#5843] [UTM Help (iOS)]
**Describe the issue**
Basically is there a version of UTM SE for TestFlight that still is open?
**Configuration (required)**
* UTM Version: SE
* OS Version: iOS 17.1
* Device Model: iPad 7th generation
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? N/A

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment]
No, Apple took it down

**Reply by Sahelalemi** [reply]
Ah ok
